PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/container/pushimage/ui/SettingPanel.java

```diff
@@ -179,6 +192,20 @@ private void setupArtifactCombo(List<Artifact> artifacts, String targetPath) {
         isCbArtifactInited = true;
     }
 
+    private void setupMavenProjectCombo(List<MavenProject> mvnprjs, String targetPath) {
```

Comment:
Can we also add module icon in the combo box item

Reply:
added